howto/install/customise.md

```diff
@@ -1,192 +1,60 @@
-
-The `--map-machine` argument for the `juju deploy` command maps the machines defined inside the bundle to those your Juju controller has registered in the model. See the [Juju documentation](https://jaas.ai/docs/charm-bundles) for more details.
-
-
-## Customising the Deployment
-
-A number of the scenarios outlined at the start of this document involved customising the Anbox Cloud install. There are two main ways to do this:
-
- 1. Using overlays in conjunction with the published Anbox Cloud bundle.
- 2. Editing the bundle file itself.
-
-Using an overlay means you can easily apply your customisation to different versions of the bundle, with the possible downside that changes in the structure of new versions of Anbox Cloud may render your overlay obsolete or non-functional (depending on what exactly your overlay does).
-
-Saving a copy of the bundle file and editing that means that your customisation will always work, but of course, requires that you create a new file for each version of Anbox Cloud.
-
-Both methods are described below.
-
-
-### Using Overlays
-
-A *bundle overlay* is a fragment of valid YAML which is dynamically merged on top of a bundle before deployment, rather like a patch file. The fragment can contain any additional or alternative YAML which is intelligible to Juju. For example, to specify custom instance types for the machines used in your cloud of choice, the following fragment could be used:
-
-```bash
-$ cat overlay.yaml
-machines:
-'0':
-  series: bionic
-  constraints: "instance-type=m4.xlarge root-disk=40G"
-'1':
-  series: bionic
-  constraints: "instance-type=m4.xlarge root-disk=40G"
-'2':
-  series: bionic
-  constraints: "instance-type=g3s.xlarge root-disk=50G"
-'3':
-  series: bionic
-  constraints: "instance-type=m4.xlarge root-disk=40G"
-```
-
-Juju's bundle format, and valid YAML are discussed more fully in the Juju documentation. In this example it merely selects a specific instance-types for the different machines.
-
-To use this overlay with the Anbox Cloud bundle, it is specified during deploy like this:
-
-```bash
-$ juju deploy anbox-cloud --overlay ua.yaml --overlay ~/path/overlay.yaml
-```
-
-Substitute in the local path and filename to point to your YAML fragment.
-
-
-#### Changing Configuration Values
-
-Configuration settings are mapped to "options" under the charm entries in the bundle YAML. Usually these are only expressed when they differ from the default value in the charm. For example, if you look at the fragment for *anbox-stream-gateway* in the *anbox-cloud* bundle:
+For example, consider the following fragment for the `anbox-stream-gateway` in the `anbox-cloud` bundle:
 
 ```yaml
 anbox-stream-gateway:
   charm: cs:~anbox-charmers/anbox-stream-gateway-23
   expose: true
   num_units: 1
   options:
-    enable_dev_ui: true
     prometheus_port: 10001
     use_insecure_tls: true
```

Comment:
```suggestion
    # We use self signed TLS certificates by default. If proper signed certificates are used this can be dropped
    use_insecure_tls: true
```